state-chain/primitives/src/lib.rs

```diff
@@ -457,3 +457,31 @@ pub struct DcaParameters {
 }
 
 pub type ShortId = u8;
+
+#[derive(RuntimeDebug, PartialEq, Eq, Encode, Decode, TypeInfo, Clone, Default)]
+pub enum CcmAuxDataLookupKey<BlockNumber: Clone> {
+	#[default]
+	NotRequired,
+	Alt {
+		// Used to lookup ALT query result
+		swap_request_id: SwapRequestId,
+		// Used to calculate when to expire and refund ccm
+		block_created: BlockNumber,
```

Comment:
nit: The type already tells what type it is (plus `block_created` sounds like a function that returns a bool)
```suggestion
		created_at: BlockNumber,
```
would be same for the function too ofc